ui/src/dashboards/containers/DashboardPage.js

```diff
@@ -22,6 +22,8 @@ import {
 } from 'shared/actions/app'
 import {presentationButtonDispatcher} from 'shared/dispatchers'
 
+const defaultTimeRange = {upper: null, lower: 'now() - 15m'}
```

Comment:
I can add the format key for sure.  Should we choose to upgrade the shape of `timeRange` this will make it much easier to parse out how to treat older shapes of `timeRange`.  

I'm going to keep the shape of `timeRange` in the "influxql" format until an engineering need presents itself for a more expressive shape.  I'll expand the `buildQuery` function to expect a "format", which will lay the ground work to expand the `timeRange` should we need to.